bitrise.yml

```diff
@@ -200,9 +200,12 @@ workflows:
     after_run:
     - _run-sonarqube-scanner
     steps:
-    - danger@2:
+    - script@1:
+        title: Danger
         inputs:
-        - github_api_token: "$DANGER_GITHUB_API_TOKEN"
+        - content: |-
+            bundle install
+            bundle exec danger --dangerfile=config/danger/Dangerfile
```

Comment:
This is nice. Maybe we could use a config submodule now on iOS too @imaginaris @apparition47 ?